.github/workflows/validate-semantic-pr.yml

```diff
@@ -36,12 +36,13 @@ jobs:
             test
             deps
           scopes: |
+            main
             helm-chart
             scheduler
             lifecycle-operator
             cert-manager
             metrics-operator
-            functions-runtime
+            deno-runtime
             python-runtime
```

Comment:
maybe we could group together the scope under just one called `runtime`

Reply:
ah i would leave them separate for now and see how it goes